optimum/intel/__init__.py

```diff
@@ -185,6 +185,7 @@
             "OVModelOpenCLIPVisual",
             "OVModelOpenCLIPText",
             "OVModelOpenCLIPForZeroShotImageClassification",
+            "OVModelForZeroShotImageClassification",
             "OVSamModel",
         ]
     )
```

Comment:
could you also add in https://github.com/huggingface/optimum-intel/blob/f53b878b98f7680902e83f869767dc62d69038ac/optimum/intel/__init__.py#L339

Reply:
added